### DISCUSSION TITLE
How can I update the meta info from front-end?

### DISCUSSION THREAD

**Discussion by leeziiAtCn** [vercel/next.js#14257] [Help]
I want to create a new <meta> , But this new <meta> is covered by next-js.
``` javascript
// front-end code
        <script
          dangerouslySetInnerHTML={{
            __html: `let metaEl = document.querySelector('meta[name="viewport"]');metaEl.setAttribute('content','width=device-width,user-scalable=no,initial-scale=1')`,
          }}
        />
```

**Comment by enricoschaaf** [comment]
You need to use a custom document to override the next default. https://nextjs.org/docs/advanced-features/custom-document

**Reply by enricoschaaf** [reply]
If you want dynamically change it after the page has loaded you can use next/head on the page you want to implement this
